data/db-room/src/main/java/io/github/droidkaigi/confsched2019/data/db/RoomDatabase.kt

```diff
@@ -92,4 +97,15 @@ class RoomDatabase @Inject constructor(
             }
         }
     }
+
+    override suspend fun staffs(): List<StaffEntity> = staffDao.allStaffs()
+
+    override suspend fun save(apiResponse: StaffResponse) {
+        withContext(coroutineContext) {
+            database.runInTransaction {
+                val staffs = apiResponse.staffs.toStaffEntities()
+                staffDao.insert(staffs)
```

Comment:
Maybe we should use "deleteAll()" before insert(). Because If someone quits the staff for some reason, it seems to remain. 👀

Reply:
That's a good catch 💯